components/style/config.rs

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /*
+ * servo_config is a crate in components/config that pulls in a lot of random stuff. In gecko mode,
+ * most of this is unnecessary. However, we pull this in for Stylo because the style crate needs
+ * to check if certain options are set. This module exposes servo_config things in Servo mode, and
+ * exposes dummy functions that just return a constant boolean in Gecko mode. */
+
+#[cfg(feature = "servo")] use servo_config;
+
+// prefs functions
+#[cfg(feature = "servo")] #[allow(missing_docs)]
```

Comment:
nit: I'm not comfortable with  these `#[allow(missing_docs)]` attributes but it's OK, I guess. At least we can move them to a separate line maybe?